### PR TITLE
test: Temporarily permit old/new seal formats in timelines

### PR DESCRIPTION
**Pull request by JLGarber**
Per <https://github.com/quisquous/cactbot/pull/3673#issuecomment-980494383>, this should permit passing tests for all timeline seal/unseal entries that use the new log line format. Note the error message doesn't include the old format, since we shouldn't be using it for any new work.

### CODE REVIEW

File: test/helper/test_timeline.js [hunk 1]
```diff
@@ -303,15 +303,19 @@ const testTimelineFiles = (timelineFiles) => {
...
             const regex = sync.regex.source;
+            // FIXME: This test will currently accept log lines with or without a second colon,
+            // as "0839:" or "0839::".
+            // Once we have completely converted things for 6.0,
+            // we should come back here and make the doubled colon non-optional.
             if (regex.includes('is no longer sealed')) {
               assert(
-                regex.includes('00:0839:.*is no longer sealed'),
-                `${timelineFile}:${sync.lineNumber} 'is no longer sealed' sync must be exactly '00:0839:.*is no longer sealed'`,
+                /00:0839::?\.\*is no longer sealed/.exec(regex),
```
Comment (quisquous) [inline test/helper/test_timeline.js:312]:
```suggestion
                /00:0839::?.*is no longer sealed/.exec(regex),
```

Comment (JLGarber) [reply test/helper/test_timeline.js:312]: Oh, that's fair, I guess there might be edge cases where we do want to permit specific unseal lines, as opposed to the One True Normalized one.

Comment (quisquous) [reply test/helper/test_timeline.js]: Oh oh oh, you're right.  I think it should be like it was before I made a mistaken edit.  Sorry, I shouldn't send code review comments when i'm tired.

Comment (JLGarber) [reply test/helper/test_timeline.js:312]: It is a *good* thing for all of us that tomorrow is Saturday and not Monday!
